ini/ini_map.ts

```diff
@@ -388,34 +380,23 @@ export class IniMap {
   }
 
   *#readTextLines(text: string): Generator<string> {
-    const lineBreak = "\r\n";
     const { length } = text;
-    let lineBreakLength = -1;
     let line = "";
 
     for (let i = 0; i < length; i += 1) {
       const char = text[i]!;
 
-      if (lineBreak.includes(char)) {
+      if (char === "\n" || char === "\r") {
         yield line;
         line = "";
-        if (lineBreakLength === -1) {
-          const ahead = text[i + 1];
-          if (
-            ahead !== undefined && ahead !== char && lineBreak.includes(ahead)
-          ) {
-            if (!this.#formatting.lineBreak) {
-              this.#formatting.lineBreak = char + ahead;
-            }
-            lineBreakLength = 1;
-          } else {
-            if (!this.#formatting.lineBreak) {
-              this.#formatting.lineBreak = char;
-            }
-            lineBreakLength = 0;
+        if (char === "\r" && text[i + 1] === "\n") {
+          i++;
+          if (!this.#formatting.lineBreak) {
+            this.#formatting.lineBreak = "\r\n";
           }
+        } else if (!this.#formatting.lineBreak) {
+          this.#formatting.lineBreak = char;
         }
-        i += lineBreakLength;
```

Comment:
This code above was previously incorrectly identifying stuff like `\n\r` as a single line break when it should be two.